app/models/location_report.rb

```diff
@@ -134,18 +146,29 @@ def plates_list
       end
   end
 
+  def received_date(cur_plate)
+    cur_plate
+      &.asset_audits
+      &.where(key: 'slf_receive_plates')
+      &.where('message LIKE ?', '%Reception fridge%')
+      &.first
+      &.created_at
+      &.strftime('%Y-%m-%d %H:%M:%S')
+  end
```

Comment:
Could be worth a question to Danni - if it has been 'received' multiple times, whether to use the first or the last event. Although hopefully it's an edge case.